app/components/Admin/AddIntake.tsx

```diff
@@ -171,11 +230,28 @@ const AddIntake: React.FC<Props> = ({ applicationQuery }) => {
       errors?.endDate.addError('End date & time must be entered');
     }
 
-    if (startDateTime <= currentDateTime) {
+    if (isEdit && nextIntake && endDateTime >= nextIntakeStartDateTime) {
+      errors?.endDate.addError(
+        'End date & time must not overlap with the next intake'
+      );
+    }
+
+    if (
+      (!isEdit && startDateTime <= currentDateTime) ||
+      // only check start date is in the future if editing an intake that has not started yet and does not have start date locked
+      (isEdit &&
+        currentDateTime <= intakeStartDate &&
+        startDateTime <= currentDateTime)
```

Comment:
As for simplifying the logic, maybe capturing this in a variable to make it more readable? But otherwise I'm not sure where we can simplify this.